packages/search/src/elasticsearch/dbhelper.test.ts

```diff
@@ -9,30 +9,53 @@
  * Copyright (C) The OpenCRVS Authors. OpenCRVS and the OpenCRVS
  * graphic logo are (registered/a) trademark(s) of Plan International.
  */
+// @ts-ignore
 import {
   indexComposition,
   updateComposition,
   searchForDuplicates,
   searchByCompositionId
 } from '@search/elasticsearch/dbhelper'
 import { mockCompositionBody } from '@search/test/utils'
-import { client } from '@search/elasticsearch/client'
 import { logger } from '@search/logger'
 import { IBirthCompositionBody } from '@search/elasticsearch/utils'
+import * as elasticsearch from '@elastic/elasticsearch'
+import { StartedElasticsearchContainer } from 'testcontainers'
+// @ts-ignore
+import {
+  startContainer,
+  stopContainer
+} from '@search/features/registration/deduplicate/elasticSearchTestContainer'
+
+jest.setTimeout(10 * 60 * 1000)
 
 describe('elasticsearch db helper', () => {
   let indexSpy: jest.SpyInstance<any, any[]>
   let updateSpy: jest.SpyInstance<any, any[]>
   let searchSpy: jest.SpyInstance<any, any[]>
+  let container: StartedElasticsearchContainer
+  let client: elasticsearch.Client
 
   describe('elasticsearch db helper', () => {
-    beforeAll(() => {
+    beforeAll(async () => {
       logger.error = jest.fn()
+
+      container = await startContainer()
+      const host = container?.getHost() ?? '0.0.0.0'
+      const port = container?.getMappedPort(9200) ?? 9200
+
+      client = new elasticsearch.Client({
+        node: `http://${host}:${port}`
+      })
+    })
+
+    it('should check elasticsearch is up', async () => {
+      await client.ping()
     })
 
     it('should index a composition with proper configuration', async () => {
       indexSpy = jest.spyOn(client, 'index')
-      indexComposition('testId', mockCompositionBody)
+      await indexComposition('testId', mockCompositionBody, client)
```

Comment:
These awaits shouldn't be needed here as the test only checks if the spied function `index` is called but the test doesn't expect any result